entries/i/id.muni.cz.json

```
@@ -0,0 +1,18 @@
+{
+  "MUNI Unified Login": {
+    "domain": "id.muni.cz",
+    "additional-domains": [
+      "ics.id.muni.cz"
+    ],
```

We don't use subdomains. Please rename the files to `muni.cz.json` and `muni.cz.svg`
```suggestion
    "domain": "muni.cz",
```

---

There are subdomain files, like [id.atlassian.com.json](https://github.com/2factorauth/twofactorauth/blob/master/entries/i/id.atlassian.com.json) and [id.nintendo.net.json](https://github.com/2factorauth/twofactorauth/blob/master/entries/i/id.nintendo.net.json). Should I change mine to `muni.cz` or leave it as it is?

---

I should have said, we don't use subdomains _any longer_. Apparently some have slipped through.